ferminet/hamiltonian.py

```diff
@@ -173,25 +174,25 @@ def potential_nuclear_nuclear(charges: Array, atoms: Array) -> jnp.ndarray:
     charges: Shape (natoms). Nuclear charges of the atoms.
     atoms: Shape (natoms, ndim). Positions of the atoms.
   """
-  r_aa = jnp.linalg.norm(atoms[None, ...] - atoms[:, None], axis=-1)
+  natoms = atoms.shape[0]
+  r_aa = jnp.sqrt(jnp.sum((atoms[None, ...] - atoms[:, None] + 
+      jnp.eye(natoms)[..., None])**2, axis=-1))
   return jnp.sum(
       jnp.triu((charges[None, ...] * charges[..., None]) / r_aa, k=1))
 
 
-def potential_energy(r_ae: Array, r_ee: Array, atoms: Array,
+def potential_energy(r_ae: Array, pos: Array, atoms: Array,
```

Comment:
Can you undo this variable rename (and other changes in this function)? Then any calls using kwargs into the library aren't broken